rx-extensions/src/main/java/com/appunite/rx/util/LogTransformer.java

```diff
@@ -34,36 +31,43 @@
 
         Logger DEFAULT = new Logger() {
             @Override
-            public void logSubscribe(@NonNull String tag, @Nonnull String observableTag) {
+            public void logOnSubscribe(@Nonnull final String tag, @Nonnull final String observableTag) {
                 final java.util.logging.Logger logger = java.util.logging.Logger.getLogger(tag);
-                logger.log(Level.INFO, observableTag + " -> onSubscribe");
+                if (logger.isLoggable(Level.INFO)) {
```

Comment:
actually I think It make sens to check because if it's not logable it will not create object `observableTag + " -> onSubscribe"`